platform/ios/src/MGLMapView.mm

```diff
@@ -3162,20 +3211,42 @@ - (void)setVisibleCoordinateBounds:(MGLCoordinateBounds)bounds edgePadding:(UIEd
 
 - (void)setVisibleCoordinates:(const CLLocationCoordinate2D *)coordinates count:(NSUInteger)count edgePadding:(UIEdgeInsets)insets animated:(BOOL)animated
 {
+    MGLLogDebug(@"Setting: %lu coordinates edgePadding: %@ animated: %@",
+                count,
+                NSStringFromUIEdgeInsets(insets),
+                MGLStringFromBOOL(animated));
     [self setVisibleCoordinates:coordinates count:count edgePadding:insets direction:self.direction animated:animated];
 }
 
 - (void)setVisibleCoordinates:(const CLLocationCoordinate2D *)coordinates count:(NSUInteger)count edgePadding:(UIEdgeInsets)insets direction:(CLLocationDirection)direction animated:(BOOL)animated
 {
+    MGLLogDebug(@"Setting: %lu coordinates edgePadding: %@ direction: %f animated: %@",
+                count,
+                NSStringFromUIEdgeInsets(insets),
+                direction,
+                MGLStringFromBOOL(animated));
     [self setVisibleCoordinates:coordinates count:count edgePadding:insets direction:direction duration:animated ? MGLAnimationDuration : 0 animationTimingFunction:nil];
 }
 
 - (void)setVisibleCoordinates:(const CLLocationCoordinate2D *)coordinates count:(NSUInteger)count edgePadding:(UIEdgeInsets)insets direction:(CLLocationDirection)direction duration:(NSTimeInterval)duration animationTimingFunction:(nullable CAMediaTimingFunction *)function {
+    MGLLogDebug(@"Setting: %lu coordinates edgePadding: %@ direction: %f duration: %f animationTimingFunction: %@",
+                count,
+                NSStringFromUIEdgeInsets(insets),
+                direction,
+                duration,
+                function);
     [self setVisibleCoordinates:coordinates count:count edgePadding:insets direction:direction duration:duration animationTimingFunction:function completionHandler:NULL];
 }
 
 - (void)setVisibleCoordinates:(const CLLocationCoordinate2D *)coordinates count:(NSUInteger)count edgePadding:(UIEdgeInsets)insets direction:(CLLocationDirection)direction duration:(NSTimeInterval)duration animationTimingFunction:(nullable CAMediaTimingFunction *)function completionHandler:(nullable void (^)(void))completion
 {
+    MGLLogDebug(@"Setting: %lu coordinates edgePadding: %@ direction: %f duration: %f animationTimingFunction: %@ completionHandler: %@",
+                count,
+                NSStringFromUIEdgeInsets(insets),
+                direction,
+                duration,
+                function,
+                completion);
```

Comment:
Nit: Given that these parameters have no argument names (as they would with a similarly indented ObjC method) and that the method name just above it is quite long, this indentation (and others in this file) doesn’t seem useful to me.